### ISSUE TITLE
Grafana: need to click before scrolling is working

### ISSUE THREAD

**Issue by manio** [philc/vimium#4029]
Hi!
I am using grafana and cannot scroll down using vimium. First I have to click somewhere in the middle of the page, and only after this I can scroll using `j`, `d`

Steps to reproduce:
1. Go here: https://play.grafana.org/d/000000012/grafana-play-home?orgId=1
2. Try to scroll down. [not working]
3. Click in the middle of the page
4. Try to scroll again [working]

`gf` and `gF` doesn't help

**Comment by gdh1995** [comment]
Duplicate of #2372 and #3760. You may see https://github.com/philc/vimium/issues/3760#issuecomment-812882124 for the solution.

**Comment by manio** [comment]
Much thanks, closing.
